### PR TITLE
Pioneer 15 Leg 2 updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates for Pioneer 15 Leg 2:
CNSM-12 recovery updates
OSSM-12 deployment and OSSM-11 recovery updates (includes 1 sensor update - NSIF FLORTD from 1214 to 1216)
PMUO-16 deployment and PMUO-15 recovery updates
Gliders 376 and 379 recovery updates

QLR draft pdf attached.  Deployment & recovery written logs and written configs have been posted to Vault in Operations > Coastal Pioneer Cruises > Pioneer 15 Deployment > Pioneer 15 Logs

[3204-01502_Quick_Look_Cruise_Report_Coastal_Pioneer_15_2020-11-09b.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/5519766/3204-01502_Quick_Look_Cruise_Report_Coastal_Pioneer_15_2020-11-09b.pdf)

### CODE REVIEW

File: deployment/CP04OSSM_Deploy.csv [hunk 1]
```diff
@@ -309,65 +309,65 @@ AR34,,AR39,,CP04OSSM-SBD11-06-METBKA000,10,1,2019-04-05T14:55:00,2019-09-28T12:3
...
+AR48,,,,CP04OSSM-RID27-04-DOSTAD000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-DOSTAD-00391,39.939883,-70.876693,,7,443,
+AR48,,,,CP04OSSM-SBC11-00-CPMENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-BCPM01-50002,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD11-00-DCLENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-BDCL11-50002,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD11-01-MOPAK0000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-MOPAK0-44005,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD11-02-HYDGN0000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-HYDGN0-46156,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD11-06-METBKA000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-METLGR-00038,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD12-00-DCLENG000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGCON-BDCL12-50002,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD12-03-HYDGN0000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-HYDGN0-41552,39.939883,-70.876693,,0,443,
+AR48,,,,CP04OSSM-SBD12-04-PCO2AA000,12,1,2020-11-04T15:33:00,,CGMCP-04OSSM-00012,,CGINS-PCO2AA-35260,39.939883,-70.876693,,0,443,
```
Comment (reedan88) [inline deployment/CP04OSSM_Deploy.csv:373]: Looks good but check the FLORT S/N swapout (I think its correct).

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 2]
```diff
@@ -65,9 +65,9 @@ TS20200301,,TS20200522,,CP05MOAS-GL376-02-FLORTM000,11,1,2020-03-02T09:46:04,202
...
+TS20200819,,AR48,,CP05MOAS-GL376-00-ENG000000,12,1,2020-08-19T17:11:00,2020-11-04T20:08:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,AR48,,CP05MOAS-GL376-01-ADCPAM000,12,1,2020-08-19T17:11:00,2020-11-04T20:08:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-49696,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,AR48,,CP05MOAS-GL376-02-FLORTM000,12,1,2020-08-19T17:11:00,2020-11-04T20:08:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03116,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,AR48,,CP05MOAS-GL376-03-CTDGVM000,12,1,2020-08-19T17:11:00,2020-11-04T20:08:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09089,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,AR48,,CP05MOAS-GL376-04-DOSTAM000,12,1,2020-08-19T17:11:00,2020-11-04T20:08:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00543,39.88518,-70.39578,,1000,1000,SS-1
+TS20200819,,AR48,,CP05MOAS-GL376-05-PARADM000,12,1,2020-08-19T17:11:00,2020-11-04T20:08:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50167,39.88518,-70.39578,,1000,1000,SS-1
```
Comment (reedan88) [inline deployment/CP05MOAS-GL376_Deploy.csv:73]: Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 3]
```diff
@@ -53,9 +53,9 @@ TS20200301,,TS20200522,,CP05MOAS-GL379-02-FLORTM000,9,1,2020-03-02T09:44:16,2020
...
+TS20200819,,AR48,,CP05MOAS-GL379-00-ENG000000,10,1,2020-08-19T17:18:00,2020-11-04T20:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,AR48,,CP05MOAS-GL379-01-ADCPAM000,10,1,2020-08-19T17:18:00,2020-11-04T20:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-50865,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,AR48,,CP05MOAS-GL379-02-FLORTM000,10,1,2020-08-19T17:18:00,2020-11-04T20:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,AR48,,CP05MOAS-GL379-03-CTDGVM000,10,1,2020-08-19T17:18:00,2020-11-04T20:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,AR48,,CP05MOAS-GL379-04-DOSTAM000,10,1,2020-08-19T17:18:00,2020-11-04T20:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00171,39.88515,-70.39692,,1000,1000,FZ-1
+TS20200819,,AR48,,CP05MOAS-GL379-05-PARADM000,10,1,2020-08-19T17:18:00,2020-11-04T20:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.88515,-70.39692,,1000,1000,FZ-1
```
Comment (reedan88) [inline deployment/CP05MOAS-GL379_Deploy.csv:61]: Looks good.

**Review by reedan88 — APPROVED** [review]
Everything looks good to me.

File: deployment/CP01CNSM_Deploy.csv [hunk 4]
```diff
@@ -373,40 +373,40 @@ AR34,,AR39,,CP01CNSM-SBD12-04-PCO2AA000,11,1,2019-04-06T14:35:00,2019-09-26T17:1
...
+AR39,,AR48,,CP01CNSM-SBD11-06-METBKA000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-METLGR-00010,40.1332,-70.7783,,0,133,
+AR39,,AR48,,CP01CNSM-SBD12-00-DCLENG000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGCON-BDCL12-20001,40.1332,-70.7783,,0,133,
+AR39,,AR48,,CP01CNSM-SBD12-03-HYDGN0000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-HYDGN0-Z5971,40.1332,-70.7783,,0,133,
+AR39,,AR48,,CP01CNSM-SBD12-04-PCO2AA000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-PCO2AA-34222,40.1332,-70.7783,,0,133,
+AR39,,AR48,,CP01CNSM-SBD12-05-WAVSSA000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-WAVSSA-05791,40.1332,-70.7783,,0,133,
+AR39,,AR48,,CP01CNSM-SBD12-06-METBKA000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-METLGR-00017,40.1332,-70.7783,,0,133,
+AR39,,AR48,,CP01CNSM-SBD12-08-FDCHPA000,12,1,2019-09-27T18:30:00,2020-11-06T13:17:00,CGMCP-01CNSM-00012,,CGINS-FDCHPA-44902,40.1332,-70.7783,,-5,133,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/CP01CNSM_Deploy.csv:409]: Looks good.

File: deployment/CP02PMUO_Deploy.csv [hunk 5]
```diff
@@ -125,21 +125,21 @@ AR39,,AR44,,CP02PMUO-WFP01-02-DOFSTK000,14,1,2019-10-14T18:09:00,2020-06-09T16:0
...
+AR48,,,,CP02PMUO-SBS01-00-STCENG000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016,,CGCON-BSTC01-00005,39.942287,-70.779878,,0,453,
+AR48,,,,CP02PMUO-SBS01-01-MOPAK0000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016,,CGINS-MOPAK0-11358,39.942287,-70.779878,,0,453,
+AR48,,,,CP02PMUO-WFP01-00-WFPENG000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,39.942287,-70.779878,,429,453,
+AR48,,,,CP02PMUO-WFP01-01-VEL3DK000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,39.942287,-70.779878,,429,453,
+AR48,,,,CP02PMUO-WFP01-02-DOFSTK000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,39.942287,-70.779878,,429,453,
+AR48,,,,CP02PMUO-WFP01-03-CTDPFK000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,39.942287,-70.779878,,429,453,
+AR48,,,,CP02PMUO-WFP01-04-FLORTK000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,39.942287,-70.779878,,429,453,
+AR48,,,,CP02PMUO-WFP01-05-PARADK000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016,CGVEH-CWFPCP-10301,CGINS-PARADK-20610,39.942287,-70.779878,,429,453,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/CP02PMUO_Deploy.csv:145]: Looks good.

File: deployment/CP04OSSM_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/CP04OSSM_Deploy.csv:373]: Looks good. Yes FLORT 01214 was on ISSM.  FLORT 01216 is correct here.

File: deployment/CP05MOAS-GL376_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL376_Deploy.csv:73]: Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL379_Deploy.csv:61]: Looks good.

**Review by snwhite — APPROVED** [review]

**Comment by rtravis-whoi** [comment]
[Pioneer 15 AM Updates [DT-102]](https://trello.com/c/PK5rFp9p/102-pioneer-15-am-updates-dt-102)

**Pull request merged**